### PR TITLE
Update the meeting time for the Spring 2024 semester

### PR DESCRIPTION
**Pull request by trwbox**
@silichws @jsweenz Could one of you two build this and verify that the meeting time works. I can't image that it will break anything, and works locally for me. But want someone else to make sure before publishing the updated time for this semester.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -21,11 +21,6 @@ youtube_channel: UC-rLOtGfHGx9N1mlNNEwJNQ
 discord: https://discord.gg/3xxec7V5zN
 remote_theme: pages-themes/hacker@v0.2.0
 
-sponsors:
```
Comment (silichws) [inline _config.yml:24]: Are we not showing sponsors anymore? Also, if we are, you should also delete the actual file that this was linked to before

Comment (silichws) [reply _config.yml:24]: Oh wait, I see this is just the config. Disregard that.

**Comment by silichws** [comment]
> @silichws @jsweenz Could one of you two build this and verify that the meeting time works. I can't image that it will break anything, and works locally for me. But want someone else to make sure before publishing the updated time for this semester.

Looks good to me

**Review by silichws — APPROVED** [review]

File: _config.yml (hunk 1 above)
Comment (trwbox) [reply _config.yml:24]: Yeah this is used to auto populate the sponsors at the bottom of each page, and it was doubled up for some reason? The site did not appear to change when I removed it, I should have probably added that info to the commit comment too.

**Pull request merged**